sed/loader/sxp/loader.py

```diff
@@ -0,0 +1,891 @@
+# pylint: disable=duplicate-code
+"""
+This module implements the SXP data loader.
+This loader currently supports the SXP momentum microscope instrument.
+The raw hdf5 data is combined and saved into buffer files and loaded as a dask dataframe.
+The dataframe is a amalgamation of all h5 files for a combination of runs, where the NaNs are
+automatically forward filled across different files.
+This can then be saved as a parquet for out-of-sed processing and reread back to access other
+sed funtionality.
+Most of the structure is identical to the FLASH loader.
```

Comment:
I think just doing a replace all of flash to sxp would be good

Reply:
I put this there on purpose ;) ...

Reply:
this is the only ocurrence, btw